src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/TableWidget.tsx

```diff
@@ -87,106 +37,50 @@ const TableWidget = React.forwardRef<ChartKitWidgetRef | undefined, TableWidgetP
             onLoad,
```

Comment:
Maybe it's time to call this props smth like `onRender`? :)

Reply:
You need to switch to all widgets at once :(
Or you will have to write mapping for some types of widgets

Reply:
I suggest to rename only